### ISSUE TITLE
ARP requests not passed to OF controller

### ISSUE THREAD

**Issue by mwutzke** [NorthboundNetworks/ZodiacFX#94]
As highlighted on the Faucet mailing list [1], there appears to be a recent regression in the ZodiacFX OpenFlow implementation (from 0.84). I'm not 100% sure what part of the pipeline is causing the issue, but it seems related to ARP Req from attached hosts, that are to be processed by the Faucet controller (eg ARP for the next hop router that is provided by Faucet).

Using the Faucet configuration in [2], the symptoms appear to be:
- hosts are not able to resolve the ARP for their next hop router
- Faucet controller does not receive ARP Req frames
- The OpenFlow connection is terminated / restarted on a regular basis as seen by these Faucet logs:
```
Jun 28 23:21:06 faucet.valve INFO     DPID 123917682135978 (0x70b3d56cd3aa) sw1 Cold start configuring DP
Jun 28 23:21:06 faucet.valve INFO     DPID 123917682135978 (0x70b3d56cd3aa) sw1 Port 1 (host1 network namespace) configured
Jun 28 23:21:06 faucet.valve INFO     DPID 123917682135978 (0x70b3d56cd3aa) sw1 Port 2 (host2 network namespace) configured
Jun 28 23:21:06 faucet.valve INFO     DPID 123917682135978 (0x70b3d56cd3aa) sw1 Configuring VLAN vlan100 vid:100 untagged: Port 1
Jun 28 23:21:06 faucet.valve INFO     DPID 123917682135978 (0x70b3d56cd3aa) sw1 Configuring VLAN vlan150 vid:150 untagged: Port 2
```

This issue appears to be present in the 0.84 and 0.85 ZodiacFX releases, but with the 0.83 release the ARP operation appears to work correctly.

[1] https://lists.geant.org/sympa/arc/faucet-users/2019-06/msg00076.html
[2] https://lists.geant.org/sympa/arc/faucet-users/2019-06/msg00041.html

**Comment by mwutzke** [comment]
My investigations has indicated that there is a change in the ZodiacFX handling of unimplemented fields in the 0.84 release. As Faucet ARP support uses the OFPXMT_OFB_ARP_TPA match, this change in unimplemented fields from v0.83 to v0.84, has resulted in certain flows no longer being (incorrectly) matched.

I believe the OF controller connection reset is more closely related to #84, and should be handled under that issue.

**Comment by pzanna** [comment]
This has now been fixed in v0.86
